src/sagemaker_sklearn_extension/preprocessing/encoders.py

```diff
@@ -513,11 +533,14 @@ class RobustOrdinalEncoder(OrdinalEncoder):
 
     """
 
-    def __init__(self, categories="auto", dtype=np.float32, unknown_as_nan=False):
+    def __init__(self, categories="auto", dtype=np.float32, unknown_as_nan=False, threshold=1, max_categories=np.inf):
         super(RobustOrdinalEncoder, self).__init__(categories=categories, dtype=dtype)
         self.categories = categories
         self.dtype = dtype
         self.unknown_as_nan = unknown_as_nan
+        self.threshold = threshold
+        self.max_categories = max_categories
+        self.features_completely_under_threshold = []
```

Comment:
Let's rename this to `self.features_under_threshold_` and move to be set in the `fit` method because it's not an init parameter